src/main/cpp/subsys/SwerveChassis.cpp

```diff
@@ -299,11 +302,18 @@ void SwerveChassis::Drive
                 m_brState.angle = UpdateForPolarDrive(currentPose, goalPose, Transform2d(m_backRightLocation, m_brState.angle), chassisSpeeds);
            }
 
+            if(m_hold)

```

Comment:
I think we have to break this out as a new option; we don't want auton paths locking the wheels when it choses toward goal option.   Even deviating from the path that is above will cause issues, I believe.